processors/zipkin/src/main/java/org/hawkular/apm/processor/zipkin/TraceCompletionTimeDeriver.java

```diff
@@ -70,9 +70,9 @@ public CompletionTime processOneToOne(String tenantId, Span item) throws RetryAt
                 ct.setEndpointType("Unknown");
             }
 
-            ct.setDuration(TimeUnit.MILLISECONDS.convert(item.getDuration(), TimeUnit.NANOSECONDS));
+            ct.setDuration(TimeUnit.MILLISECONDS.convert(item.getDuration(), TimeUnit.MICROSECONDS));
```

Comment:
We should consider changing duration unit micro or nano. What if a trace completes in micro/nano seconds?